packages/nestjs-class-validators/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "nestjs-class-validators",
```

```suggestion
  "name": "@algoan/nestjs-class-validators",
```

It also should be added it here:
https://github.com/algoan/nestjs-components/blob/5e327217a30fe766eb12b214620a9bd19aa14974/package.json#L91-L92